src/qtbase/QSizeF.js

```diff
@@ -0,0 +1,7 @@
+class QSizeF extends QObject {
+  constructor( width, height ) {
+    super();
+    createProperty("real", this, "width", { initialValue : width });
```

Comment:
Single (`'`) quotes are slightly more common in the sources than double (`"`). I am not yet sure if we would want to unify them to a single format (i.e. single vs double), but in case if we do, I don't see a reason for adding the new code with the less popular one =).

Could you replace the quotes in the newly added js files to single (`'`), please?